pkgs/applications/editors/vscode/extensions/rescript/rescript-editor-analysis.nix

```
@@ -1,21 +1,31 @@
-{ lib, stdenv, fetchFromGitHub, bash, ocaml }:
+{ lib, stdenv, fetchFromGitHub, bash, ocaml, dune_3, extensionVersion }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, bash, ocaml, dune_3, version }:
```
and the other variables respectively